### DISCUSSION TITLE
images are look so crapy any idea how to fix this

### DISCUSSION THREAD

**Discussion by tuhinadri** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#43] [Q&A] [answered]
Hey, can you tell me how has this happened! It's not so high-quality image like TMDB's!
![image](https://user-images.githubusercontent.com/57254086/111298858-457c4600-8675-11eb-9f98-c8a61490d829.png)

**Comment by DirtyRacer1337** [comment]
I noticed that. I happened after upgrade on 10.7.0.

I checked, It's loading image with url like `http://localhost:8096/Items/88414fe6d24f0e08a28e5b6de4b9c232/Images/Primary?maxHeight=320&maxWidth=213&tag=6ab9b3e159572d5ac39335243035b2c2&quality=90`
It's have `maxHeight` and `maxWidth`, if I delete that parameters, Images looks much better.

Also, I noticed, It's happened only when `height < width`, maybe it's error.

Not every site have acceptable poster, I use that image that on site.

**Reply by DirtyRacer1337** [reply]
Solution is swap the parameters values if height < width

**Reply by tuhinadri** [reply]
how to do that?

**Reply by tuhinadri** [reply]
where is the parameter by the way? & can we not increase the size of max-width & max-height

**Reply by tuhinadri** [reply]
definitely, this will be a CSS trick! https://jellyfin.org/docs/general/clients/css-customization.html could you please help me!

**Reply by DirtyRacer1337** [reply]
I think It's should be somewhere in [Jellyfin Web](https://github.com/jellyfin/jellyfin-web). CSS not help, server gives wrong sized image by JavaScript request.

**Reply by tuhinadri** [reply]
how did you do?

**Reply by DirtyRacer1337** [reply]
I only tested, I haven't working solution

**Reply by tuhinadri** [reply]
oooooooo! ok! I post it on r/jellyfin on that! let see!

**Reply by DirtyRacer1337** [reply]
I mean not css parameters, I mean HTTP GET parameters

localhost:8096/Items/88414fe6d24f0e08a28e5b6de4b9c232/Images/Primary?**maxHeight**=320&**maxWidth**=213&tag=6ab9b3e159572d5ac39335243035b2c2&quality=90

Example: Image have size `1920x1080`, but it tries to load `213x320` which gives `213x120`, instead `320x213` which gives `320x180`
It's web ui error

**Reply by tuhinadri** [reply]
Yes! exactly! I think they thought peoples only used Jelly for ripping Movie collections, but then we entered!

**Reply by tuhinadri** [reply]
![image](https://user-images.githubusercontent.com/57254086/111472211-c0b22a80-874f-11eb-9a51-ebf17881931c.png)
they send me this link also! https://github.com/jellyfin/jellyfin/pull/5556
so i have to find the file?

**Reply by DirtyRacer1337** [reply]
Good that someone solving this issue. I still think problem in default aspects ratio = 2/3, instead calculating actual, but anyway I made hot dirty fix which delete image size reduction, until they fix it and release it. 

Images loading slower, only good for local server, but better than crappy image.

Based on [5fb38139b86c3c5bf21704abc47b7a296c9a3056 ](https://github.com/jellyfin/jellyfin-web/commit/5fb38139b86c3c5bf21704abc47b7a296c9a3056)
[jellyfin-web.zip](https://www.mediafire.com/file/inhe1wjoyr3n0zz/jellyfin-web.zip/file)

or like it was before 2x images
[jellyfin-web.zip](https://www.mediafire.com/file/fwxdr3nawsdjf4l/jellyfin-web.2x-images.zip/file)

**Reply by tuhinadri** [reply]
BTW I have one question Suppose I have one video from a Network like Brazzers Exxtra! So in this case what should I type?
1. Brazzersexxtra - 2018-12-11 - The Real Thing.mp4
or 
2. Brazzers Exxtra - 2018-12-11 - The Real Thing.mp4
or
3. Exxtra - 2018-12-11 - The Real Thing.mp4

![image](https://user-images.githubusercontent.com/57254086/112468348-2759c880-8d8e-11eb-8c32-9ec05a9e5995.png)
which one?

**Reply by tuhinadri** [reply]
Nothing is working for me again! don't what's happening! not just brazzers!

**Reply by tuhinadri** [reply]
![image](https://user-images.githubusercontent.com/57254086/112482836-77d92200-8d9e-11eb-9a10-af47b0dd14c9.png)

**Reply by tuhinadri** [reply]
ok that's the main problem!

**Reply by tuhinadri** [reply]
I deleted the trigger now they are working! but hope It's not a mistake

**Reply by DirtyRacer1337** [reply]
Probably update fixed in 0740923098ad165c0aea07585dda7f304f1064eb and fixed sitelist in f1463053532faac69918af50110efb1edbecac89

**Reply by tuhinadri** [reply]
1. how to update it! or is it auto-update?
2.  wicked not working
3.  how to manually identify data if it's fetched wrong!

**Reply by tuhinadri** [reply]
BTW how to name Jav movies? any example! They have only id like
 pred00251
 & some time only 
 pred251 
 or
 pred-251
 
 which one could phoenix do better?

**Reply by tuhinadri** [reply]
I'm looking from metadata from r18

**Reply by DirtyRacer1337** [reply]
1. use [nighty build](https://github.com/DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult/tree/nightly)
2. it's not in sitelist, but I added it in f8bbff03e8d43dbfa47f1dcde00a3a9707a0f34d, use `update database`
3. use identify function
4. better like `pred-251` or `pred 251`, just should be some symbol between, but it's should works anyway if site name in beginning of name.

**Comment by DirtyRacer1337** [comment, marked as answer]
I tested it and it's works 
jellyfin/jellyfin#5556
